PBN RCT.Rmd

```diff
@@ -69,35 +69,144 @@ followup_data_deduplicated <- followup_data %>%
   slice(1)
 
 #Add the follow-up data to the baseline data. 
-###NEED TO CHECK THIS WORK###
-participants_with_followp <- included_participants %>%
+participants_with_followup <- included_participants %>%
   left_join(followup_data_deduplicated, by = "Participant_ID")
+```
+
+```{r clean data}
+#Make necessary variables factors
+###STILL NEED TO DO GENDER AND RACE/ETHNICITY, SEE IF I CAN DOWNLOAD DATA BETTER###
+participants_with_followup$b_dem_saab <- factor(
```

Comment:
`as.factor(participants_with_followup$b_dem_saab` should do this, I think!

You can also get all of these done at once that way:

```
participants_with_followup %>%
  mutate(
    across(
      all_of(c(b_dem_saab, etc.)),
      as.factor
    )
  )
```